feature-libs/organization/administration/components/unit/links/addresses/services/current-unit-address.service.ts

```diff
@@ -34,7 +34,9 @@ export class CurrentUnitAddressService extends CurrentItemService<Address> {
   }
 
   protected getItem(unitUid: string, addressId: string): Observable<Address> {
-    return addressId ? this.unitService.getAddress(unitUid, addressId) : of({});
+    return addressId
+      ? this.unitService.getAddress(unitUid, addressId)
+      : of(null);
```

Comment:
this was a reason of wrong translation, an empty object was passing into next method in form.component
``` 
  protected setI18nRoot(item: T): void {
    // concatenate the i18n root with .edit or .create suffix
    this.i18n = this.i18nRoot + (item ? '.edit' : '.create');
  }
```